scripts/ci/run_ci.sh

```diff
@@ -96,9 +96,18 @@ if [ -n "$MAIN_CI" ]; then
 	fi
 	source zephyr-env.sh
 	SANITYCHECK="${ZEPHYR_BASE}/scripts/sanitycheck"
+
+	# Possibly the only record of what exact version is being tested:
+	short_git_log='git log -n 5 --oneline --decorate --abbrev=12 '
+
 	if [ -n "$PULL_REQUEST_NR" ]; then
+		$short_git_log $REMOTE/${BRANCH}
```

Comment:
Nit: Could make it `$REMOTE/$BRANCH` to have a consistent style.

Reply:
This current version is consistent with... the inconsistent code 4 lines below :-( 
I will change just this line (I'd rather not fix the other line and take its git blame)